scheduler/src/cook/kubernetes/api.clj

```diff
@@ -1641,6 +1656,16 @@
                 (str "Pod name from pod (" pod-name-from-pod ") "
                      "does not match pod name argument (" pod-name ")"))
         (log/info "In" compute-cluster-name "compute cluster, launching pod with name" pod-name "in namespace" namespace ":" (.serialize json pod))
+        (let [event-map {:pod-name pod-name
+                         :namespace namespace
+                         :cluster-name compute-cluster-name
+                         :event-type passport/pod-launching}
+              instance-uuid (pod-name->instance-uuid pod-name)
+              job-uuid (pod-name->job-uuid pod-name)
+              event-map (cond
+                          instance-uuid (assoc event-map :instance-uuid instance-uuid)
+                          job-uuid (assoc event-map :job-uuid job-uuid))]
```

Comment:
Thoughts on using `cond->` (something like below)?

Nitpick: We usually sort map literal keys lexicographically.

```suggestion
        (let [instance-uuid (pod-name->instance-uuid pod-name)
              job-uuid (pod-name->job-uuid pod-name)
              event-map (cond-> 
                          {:cluster-name compute-cluster-name
                           :event-type passport/pod-launching
                           :namespace namespace
                           :pod-name pod-name}
                          instance-uuid (assoc :instance-uuid instance-uuid)
                          job-uuid (assoc :job-uuid job-uuid))]
```

Reply:
Yeah, I'll try using `cond->` like your change. Like this change also because `cond` short-circuits and `cond->` does not.